config/services.config.js

```diff
@@ -0,0 +1,44 @@
+'use strict'
+
+require('dotenv').config()
+
+const config = {
+  addressFacade: {
+    url: process.env.EA_ADDRESS_FACADE_URL
+  },
+  chargingModule: {
+    url: process.env.CHARGING_MODULE_URL
+  },
+  serviceForeground: {
+    url: process.env.SERVICE_FOREGROUND_URL
+  },
+  serviceBackground: {
+    url: process.env.SERVICE_BACKGROUND_URL
+  },
+  reporting: {
+    url: process.env.REPORTING_URL
+  },
+  import: {
+    url: process.env.IMPORT_URL
+  },
+  tacticalCrm: {
+    url: process.env.TACTICAL_CRM_URL
+  },
+  externalUi: {
+    url: process.env.EXTERNAL_UI_URL
+  },
+  internalUi: {
+    url: process.env.INTERNAL_UI_URL
+  },
+  tacticalIdm: {
+    url: process.env.TACTICAL_IDM_URL
+  },
+  permitRepository: {
+    url: process.env.PERMIT_REPOSITORY_URL
+  },
+  returns: {
+    url: process.env.RETURNS_URL
+  }
+}
+
+module.exports = config
```

Comment:
Just a thought. I think we could directly export our config instead of assigning it to `config` and then exporting it, eg:

```js
module.exports = {
  addressFacade: {
    // etc.
  }
}
```

Is this something we would consider?

Reply:
💯 I'd be up for that. But I'd go with doing this to all our config in one go. This means I'd merge this change and then add a new PR to cover refactoring all config to follow that pattern.